vmdb/app/helpers/container_helper/textual_summary.rb

```diff
@@ -13,6 +13,22 @@ def textual_group_properties
   #
 
   def textual_image
-    {:label => "image", :value => @record.image}
+    {:label => "Image", :value => @record.image}
+  end
+
+  def textual_name
+    {:label => "Name", :value => @record.name}
+  end
+
+  def textual_state
+    {:label => "State", :value => @record.state}
+  end
+
+  def textual_restart_count
+    {:label => "Restart Count", :value => @record.restart_count}
+  end
+
+  def textual_container_id
+    {:label => "Container ID", :value => @record.container_id}
```

Comment:
@erinboyd we don't display id on any of the summary screens. Tthis should display a name of container instead of id with a link that points to that container, i am not sure what container_id refers to here, is it a relationship?